src/nnpack/NNPACK.jl

```diff
@@ -0,0 +1,26 @@
+include("libnnpack_types.jl")
+include("error.jl")
+include("libnnpack.jl")
+
+const depsjl_path = joinpath(dirname(@__FILE__), "..", "..", "deps", "deps.jl")
+if !isfile(depsjl_path)
+    error("NNPACK not installed properly, run Pkg.build(\"NNlib\"), restart Julia and try again")
+end
+include(depsjl_path)
+
+const nnlib_interface_path = joinpath(dirname(@__FILE__), "nnlib.jl")
+@init begin
+    check_deps()
+    try
+        nnp_initialize()
```

Comment:
Can we explicitly check the [status code](https://github.com/Maratyszcza/NNPACK/blob/1e005b0c2777f39972a4ac15bea03e0e315a3d92/include/nnpack.h#L80) rather than having a generic try/catch; otherwise load errors when including the wrappers will get hidden.